files/csscritic/update.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "npm",
+  "name": "csscritic",
+  "repo": "cburgmer/csscritic",
+  "files": {
+    "basePath": "dist/",
```

Comment:
I dont see a dist/ folder in the repo https://github.com/cburgmer/csscritic/tree/0.6.0